pkgs/applications/display-managers/sddm/default.nix

```diff
@@ -1,48 +1,32 @@
-{ mkDerivation, lib, fetchFromGitHub, fetchpatch
-, cmake, extra-cmake-modules, pkg-config, qttools
-, libxcb, libXau, pam, qtbase, qtdeclarative, qtquickcontrols2, systemd, xkeyboardconfig
+{ stdenv, lib, fetchFromGitHub
+, cmake, pkg-config, qttools
+, libxcb, libXau, pam, qtbase, wrapQtAppsHook, qtdeclarative
+, qtquickcontrols2 ? null, systemd, xkeyboardconfig
```

Comment:
Why the null? Does it not exist for qt6? Is there a nicer way to solve this?

Reply:
In 6, QQC2 is part of qtbase.

Reply:
maybe qtdeclarative? qtbase doesn't have QML stuff

Reply:
Yeah, brainfart.